VAMobile/e2e/tests/AppointmentsExpanded.e2e.ts

```diff
@@ -20,43 +18,47 @@ export const Appointmentse2eConstants = {
   APPOINTMENT_CANCEL_REQUEST_TEXT: device.getPlatform() === 'ios' ? 'Cancel Request' : 'Cancel Request ',
 }
 
-const checkUpcomingApptDetails = async (appointmentType, appointmentStatus, pastAppointment = false) => {
+const checkUpcomingApptDetails = async (
+  appointmentType: string,
+  appointmentStatus: string,
+  pastAppointment = false,
+) => {
   if (!pastAppointment) {
-    if (appointmentStatus == 'Confirmed') {
+    if (appointmentStatus === 'Confirmed') {
       await expect(element(by.id('addToCalendarTestID'))).toExist()
       await expect(element(by.id('upcomingApptCancellationTestID'))).toExist()
-    } else if (appointmentStatus == 'Canceled') {
+    } else if (appointmentStatus === 'Canceled') {
       await expect(
         element(by.text('To schedule another appointment, please visit VA.gov or call your VA medical center.')),
       ).toExist()
-    } else if (appointmentStatus == 'Pending') {
+    } else if (appointmentStatus === 'Pending') {
       await expect(element(by.text('Preferred type of appointment'))).toExist()
-      if (appointmentType != 'Claim') {
+      if (appointmentType !== 'Claim') {
         await expect(element(by.text('Cancel request'))).toExist()
       }
     }
   }
 
-  if (appointmentType == 'Onsite') {
+  if (appointmentType === 'Onsite') {
     await expect(element(by.text('VA Video Connect\r\nVA location'))).toExist()
     await expect(element(by.id('directionsTestID'))).toExist()
     await expect(element(by.id('CallVATestID')).atIndex(0)).toExist()
     await expect(element(by.id('CallTTYTestID')).atIndex(0)).toExist()
-  } else if (appointmentType == 'ATLAS') {
+  } else if (appointmentType === 'ATLAS') {
     await expect(element(by.text('VA Video Connect\r\nATLAS location'))).toExist()
     await expect(element(by.id('directionsTestID'))).toExist()
   } else if (appointmentType === 'Home') {
     await expect(element(by.text('VA Video Connect\r\nHome'))).toExist()
   } else if (appointmentType === 'GFE') {
     await expect(element(by.text('VA Video Connect\r\nusing a VA device'))).toExist()
-  } else if (appointmentType == 'VA' && !pastAppointment) {
+  } else if (appointmentType === 'VA' && !pastAppointment) {
     await expect(element(by.text('In-person appointment'))).toExist()
-  } else if (appointmentType == 'Claim') {
+  } else if (appointmentType === 'Claim') {
     await expect(element(by.text('Claim exam'))).toExist()
     await expect(element(by.id('directionsTestID'))).toExist()
     await expect(element(by.id('CallVATestID'))).toExist()
     await expect(element(by.id('CallTTYTestID'))).toExist()
-  } else if (appointmentType == 'Covid') {
+  } else if (appointmentType === 'Covid') {
```

Comment:
Per Misty

Misty Milliron-Grant
  4 days ago
Check in with web on this, 
@Dylan Nienberg
. COVID-19 vaccine appt should be handled exactly like VA in-person appts.

These are no longer a separate type and now fall back into the VA type

Reply:
Thanks, I removed the covid-specific E2E tests to match your PR